File: cpp/dentro_fora.cpp
#include <iostream>

using namespace std;

int main(){
    int n, x, fora = 0, dentro = 0;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    for(int i=0; i<n; i++){
        cout << "Digite um numero: ";
        cin >> x;

        if(x < 10 || x > 20){
            fora++;
        }
        else{
            dentro++;
        }
    }

    cout << dentro << " DENTRO" << endl;
    cout << fora << " FORA" << endl;

    return 0;
}

File: python/dentro_fora.py
n: int; x: int; fora: int; dentro: int

n = int(input("Quantos numeros voce vai digitar? "))

fora = 0
dentro = 0

for i in range(n):
	x = int(input("Digite um numero: "))

	if x < 10 or x > 20:
		fora = fora + 1
	else:
		dentro = dentro + 1

print(f"{dentro} DENTRO")
print(f"{fora} FORA")